drkafka/scripts/run_in_container.sh

```diff
@@ -31,6 +31,4 @@ exec java -server -Xmx${HEAP_SIZE} -Xms${HEAP_SIZE}  -XX:NewSize=${NEW_SIZE} \
     -Dcom.sun.management.jmxremote -Dcom.sun.management.jmxremote.ssl=false \
     -Dcom.sun.management.jmxremote.authenticate=false -Dcom.sun.management.jmxremote.port=10102 \
     -Dfile.encoding=UTF-8 \
-    ${JAVA_MAIN} -config operator/config/doctorkafka.${STAGE_NAME}.properties  \
-     -topic brokerstats -zookeeper datazk001:2181/data07  \
-     -ostrichport 2052 -tsdhostport localhost:18261 -uptimeinseconds 86400
+    ${JAVA_MAIN} server $CONFIG_YAML
```

Comment:
will the script run properly with `${JAVA_MAIN} server $CONFIG_YAML`?

Reply:
Yes, since dropwizard based version has a different syntax for launching the app; it doesn't take command line parameters anymore. 

Also the only configuration that is valid in the current master of Dr. Kafka is -config i.e. path to config file. The rest of the options are no longer used and are instead loaded from the properties file which is still the case after dropwizard migration.

```
CommandLine commandLine = parseCommandLine(args);

    String configPath = commandLine.getOptionValue(CONFIG_PATH);
    LOG.info("configuration path : {}", configPath);

    ReplicaStatsManager.config =  new DoctorKafkaConfig(configPath);
    doctorKafka = new DoctorKafka(ReplicaStatsManager.config);
    doctorKafka.start();
```
https://github.com/pinterest/doctorkafka/blob/86b86ec50acf61448dcdf11f28b4f05883345b00/drkafka/src/main/java/com/pinterest/doctorkafka/DoctorKafkaMain.java#L83